src/control/cmd/daos_agent/fabric.go

```diff
@@ -24,9 +24,9 @@ func FabricNotFoundErr(netDevClass uint32) error {
 
 // FabricInterface represents a generic fabric interface.
 type FabricInterface struct {
-	Name        string `yaml:"name"`
-	Domain      string `yaml:"domain"`
-	NetDevClass uint32 `yaml:"net_dev_class"`
+	Name        string
```

Comment:
why were the tags removed?

Reply:
I was originally thinking I was going to use that struct in the yaml file directly, but I ended up creating a corresponding FabricInterfaceConfig struct instead.